src/transformers/feature_extraction_common_utils.py

```diff
@@ -238,7 +213,7 @@ def from_pretrained(
                   huggingface.co. Valid model ids can be located at the root-level, like ``bert-base-uncased``, or
                   namespaced under a user or organization name, like ``dbmdz/bert-base-german-cased``.
                 - a path to a `directory` containing a feature extractor file saved using the
-                  :func:`~transformers.PreTrainedFeatureExtractor.save_pretrained` method, e.g.,
+                  :func:`~transformers.SequenceFeatureExtractor.save_pretrained` method, e.g.,
```

Comment:
```suggestion
                  :func:`~transformers.FeatureExtractionSavingUtilsMixin.save_pretrained` method, e.g.,
```
or adapt to the new name.